### PR TITLE
Fix DG's missing View Command implementation, update Edit Command diagrams

### PR DESCRIPTION
**Pull request by harin0826**
I realised the View Command's implementation has gone missing from somewhere and I cannot find it in any of the commits, there isn't enough time to look for it so I just quickly rewrote another one.

Edit Command's diagram has also been updated to be more coherent.

### CODE REVIEW

**Comment by harin0826** [comment]
Updated diagrams under potential enhancements while I'm at it.

File: docs/diagrams/EditCommandRunActivityDiagram.puml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    if () then ([Edited person already exist in database])
+        :throw duplicate person error message;
+    else ([else])
+        :Replace person with edited person;
+        :Set protagonist to edited person in model;
+        :Reset predicate to show all person in the model;
+    endif
+endif
+stop
+@enduml
```
Comment (anchengyang) [inline docs/diagrams/EditCommandRunActivityDiagram.puml]: add new line at EOF

File: docs/diagrams/EditSkillsActivity.puml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+if () then (["s/" prefix is present])
+    :Create a new set of skills written with "s/" prefix;
+else ([else])
+    :Copy original person's set of skills;
+    :Remove removed skills from set written with "s-/" prefix;
+    :Add newly added skills to set written with "s+/" prefix;
+endif
+:Return set of skills for EditedPerson;
+stop
+@enduml
```
Comment (anchengyang) [inline docs/diagrams/EditSkillsActivity.puml]: add new line at EOF

File: docs/diagrams/ViewSequenceDiagram.puml [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+CommandResult --> ViewCommand
+deactivate CommandResult
+
+ViewCommand --> LogicManager : result
+deactivate ViewCommand
+
+[<--LogicManager
+deactivate LogicManager
+@enduml
```
Comment (anchengyang) [inline docs/diagrams/ViewSequenceDiagram.puml]: add newline at EOF

**Review by anchengyang — APPROVED** [review]
NICE

**Pull request merged**